IdentityCore/src/network/error_handler/MSIDAADRequestErrorHandler.m

```diff
@@ -135,6 +135,16 @@ - (void)handleError:(NSError *)error
         [additionalInfo setValue:@1 forKey:MSIDServerUnavailableStatusKey];
     }
     
+    if (data && data.length > 0)
+    {
+        NSString *responseString = [[NSString alloc] initWithData:data encoding:NSUTF8StringEncoding];
+        if (![NSString msidIsStringNilOrBlank:responseString])
+        {
+            NSString *teleString = responseString.length > 10 ? [responseString substringToIndex:10] : responseString;
+            [additionalInfo setValue:teleString forKey:MSIDHTTPResponseStringKey];
+        }
+    }
+    
```

Comment:
Can we log when httpResponse is nil as well, just in case?

Reply:
If it's nil, the additionnalInfo dict won't have the key/value pair so telemetry will show nil, we don't need to set it explicitly here.